### PR TITLE
Align eslint config with other Amsterdam projects

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

**Comment by janjaap** [comment]
Which other projects is the eslint config in line with? Is there a standard that projects should adhere to? Speaking of which, did you base it on https://github.com/Amsterdam/signals-frontend/tree/develop/eslint-plugin-amsterdam by any chance?

**Comment by jonkoops** [comment]
> Which other projects is the eslint config in line with? Is there a standard that projects should adhere to? Speaking of which, did you base it on https://github.com/Amsterdam/signals-frontend/tree/develop/eslint-plugin-amsterdam by any chance?

This configuration matches that of [amsterdam-react-maps](https://github.com/Amsterdam/amsterdam-react-maps/blob/master/.eslintrc.js) and [matomo-tracker](https://github.com/Amsterdam/matomo-tracker/blob/master/.eslintrc.js). As you can see in the config it's based on Airbnb TypeScript and Prettier, together with recommended configs from various sources.

**Comment by aditudorache** [comment]
@jonkoops, @janjaap, Can we discuss this matter in a fronted meeting?

File: tsconfig.eslint.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  // Reset exclusions in order to include files with .test extension.
```
Comment (robinpiets) [inline tsconfig.eslint.json:3]: Comments are not permitted in JSON

**Review by robinpiets — CHANGES_REQUESTED** [review]

File: tsconfig.eslint.json (hunk 1 above)
Comment (jonkoops) [reply tsconfig.eslint.json:3]: This is allowed for TypeScript JSON files, see: https://github.com/microsoft/TypeScript/pull/5450

**Review by aditudorache — APPROVED** [review]

**Pull request merged**